### DISCUSSION TITLE
Connect to existing open browser, scan page and insert password if password field found

### DISCUSSION THREAD

**Discussion by scohen-examity** [go-rod/rod#925] [General]
Use-case:  A browser is already opened and sitting on a page that requires a password.  Want to have app connect to the running browser and scan the DOM, if it finds the password field and some other markers, a the password would be injected and submit button clicked.  Examples would be greatly appreciated

**Comment by ysmood** [comment]
The existing doc is sufficient for this.

Chromium cdp doesn't allow connection to existing open browser, you have to launch a browser with `--remote-debugging-port` flag and follow this #190
